app/controllers/admin/users_controller.rb

```diff
@@ -76,9 +76,13 @@ def full_delete
       @user = User.find(params[:id])
       begin
         Moderator::DeleteUser.call(user: @user)
+        link = helpers.link_to("the page", admin_users_gdpr_delete_requests_path, "data-no-instant" => true)
+        # rubocop:disable Rails/OutputSafety
         message = "@#{@user.username} (email: #{@user.email.presence || 'no email'}, user_id: #{@user.id}) " \
           "has been fully deleted." \
-          "If this is a GDPR delete, delete them from Mailchimp & Google Analytics."
+          "If this is a GDPR delete, delete them from Mailchimp & Google Analytics " \
+          " and confirm on #{link}.".html_safe
+        # rubocop:enable Rails/OutputSafety
```

Comment:
You can probably avoid `Rails/OutputSafety` and calling `helpers.link_to` by using `content_tag` here with returns a `SafeBuffer`, see https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsoutputsafety